kata-deploy/scripts/kata-deploy-docker.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# If we fail for any reason a message will be displayed
+die() {
+        msg="$*"
+        echo "ERROR: $msg" >&2
+        exit 1
+}
+
+function print_usage() {
+	echo "Usage: $0 [install/remove]"
+}
+
+function install_artifacts() {
+	echo "copying kata artifacts onto host"
+	cp -a /opt/kata-artifacts/opt/kata/* /opt/kata/
+	chmod +x /opt/kata/bin/*
+}
+
+function configure_docker() {
+	echo "configuring docker"
+
+	local storage_driver=$(docker info --format '{{json .Driver}}')
+	if [ $storage_driver == "devicemapper" ]; then
+		echo "WARNING: devicemapper not configured. Firecracker configuration of kata, kata-fc, will not work"
+	fi
+	
+	local docker_version=$(docker info --format '{{json .ServerVersion}}')
+	echo "docker version $docker_version installed; 18.06 is the latest version that supports devicemapper"
+
+	conf_file="/etc/docker/daemon.json"
+	conf_file_backup="${conf_file}.bak"
+	snippet="${conf_file}.snip"
+	tmp_file="${conf_file}.tmp"
+	cat <<EOT | tee -a "$snippet"
+{
+  "runtimes": {
+    "kata-qemu": {
+      "path": "/opt/kata/bin/kata-qemu"
```

Comment:
@egernst do we need to do this. The default kata with the associated toml will always use QEMU. Using this extra step will slow down the launch. Maybe easier to just soft link the qemu toml as the default toml for efficiency,

Reply:
/me wonders how much this'll actually slow it down?

Reply:
Ah, I see - the toml would be an interesting way -- I'm just not sure its worth the inconsistency between the two hyerpvisors.  This makes it more clear/explicit.  Let's try to measure the delta?

Reply:
Another way is to just have a kata-runtime in addition. In any case I am OK either way.

Reply:
I’m not sure this is available for crio runtimeClass. Let’s take a look as a future improvement? Could save exec time.

Reply:
But this is the docker config file and it is supported by docker.

Reply:
Right, but in 1.5, we have the wrapper already available (and shared as part of the dockerfuke between docker/cri). Let me play with this though; I’m curious how much the wrapper costs.

Reply:
fixed for docker!